packages/apm/src/integrations/tracing.ts

```diff
@@ -114,9 +124,9 @@ export class Tracing implements Integration {
 
   private static _activeTransaction?: Span;
 
-  private static _currentIndex: number = 0;
+  private static _currentIndex: number = 1;
```

Comment:
Why did this change from 0 to 1?

Reply:
@rhcarvalho it looks like `0` is a magic constant returned by `pushActivity()` in certain cases; primarily to indicate that the activity was not pushed.